website/docs/installation/ferretdb/deb.md

```diff
@@ -4,6 +4,11 @@ sidebar_position: 2
 
 # DEB package
 
+:::note
+The `.deb` package for PostgreSQL with DocumentDB extension is not currently available.
```

Comment:
I have a question, we don't have `.deb` FerretDB package right? Then why do we say `.deb` package for PostgreSQL with DocumentDB extension? :thinking:

Reply:
We have the dev and rpm package for FerretDB - just not for postgres with DocumentDB extension. That will come later. 

https://github.com/FerretDB/FerretDB/releases

Reply:
Oops apology my bad :pray: It was updated since I looked at it last time :see_no_evil: